languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,30 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_OU_QUELLES_GROUPE_NOMINAL" name="quelle ou quelles groupe nominal" default="temp_off">
+      <rule>
+          <pattern>
+              <marker>
+                  <token regexp="yes">qu(&apostrophe;)?</token>
+                  <token>elle</token>
+              </marker>
+              <token postag="N.*|J.*" postag_regexp="yes"/>
+          </pattern>
+          <message>Une erreur semble être détectée dans cette phrase. Souhaitez-vous écrire <suggestion>quelle</suggestion> ?</message>
+          <example correction="quelle">Il m'a raconté <marker>qu'elle</marker> belle journée il avait passé.</example>
+      </rule>
+      <rule>
+          <pattern>
+              <marker>
+                  <token regexp="yes">qu(&apostrophe;)?</token>
+                  <token>elles</token>
+              </marker>
+              <token postag="N.*|J.*" postag_regexp="yes"/>
+          </pattern>
+          <message>Une erreur semble être détectée dans cette phrase. Souhaitez-vous écrire <suggestion>quelles</suggestion> ?</message>
+          <example correction="quelles">Il m'a raconté <marker>qu'elles</marker> grandes aventures il souhaitait vivre.</example>
+      </rule>
```

Comment:
FPs for the second rule in the RE:

- Elles ont des ressemblances au point **qu'elles** mêmes se confondent.
- De plus, elles sont déçues en remarquant **qu'elles** seules présentent un spectacle de danse d'avant-garde.

Also, this sentence is found, but the "quelles" correction is not pertinent:

- Certains auteurs considèrent **qu'elles** présente de hauts risques pour les écosystèmes marins

The real problem here is the verb "présente(nt)". This should be found by another rule, so we need an AP for this case in this rule.

Can you re-check the rule editor for FPs and cases like the above?